docs/qcfractal/source/managers_samples.rst

```diff
@@ -0,0 +1,191 @@
+Queue Manager Example YAML Files
+================================
+
+The primary way to set up a :term:`Manager` is to setup a YAML config file.
+This page provides helpful config files which mostly can be just copied
+and used in place (filling in things like :ref:`**username** and **password** <managers_server>`
+as needed.)
+
+The full documentation of every option and how it can be used can be found in
+:doc:`the Queue Manager's API <managers_config_api>`.
+
+For these examples, the ``username`` will always be "Foo" and the ``password`` will always be "b4R"
+(which are just placeholders and not valid).
+
+SLURM Cluster, Dask Adapter
+----------------------------
+
+To start a manager with a dask :term:`Adapter`, on a SLURM cluster, consuming 1 CPU and 8 GB of ram, targeting a Fractal
+Server running on that cluster, and using the SLURM partition ``default``, save the following YAML config file:
+
+.. code-block:: yaml
+
+    common:
+     adapter: dask
+     ntasks: 1
+     ncores: 1
+     memory: 8
+
+    server:
+     fractal_uri: "localhost:7777"
+     username: Foo
+     password: b4R
+
+    manager:
+     manager_name: "My First Manager"
```

Comment:
We might want to rename this to something like `MachineName_Configuration` or similar so that it will give people a better idea of a "canonical"/useful name.